website/photos/templates/photos/album.html

```diff
@@ -11,6 +11,12 @@
         <div class="container">
             <h1 class="text-center section-title">
                 {{ album.title }}
+                {% if album.event %}
+                <a class="btn btn-primary btn-second"
+                   href={{ album.event.get_absolute_url }}>
+                   Event
+                </a>
```

Comment:
```suggestion
                <a href="{{ album.event.get_absolute_url }}"
                   class="btn btn-primary btn-first"
                   data-toggle="tooltip"
                   data-placement="top"
                   title=""
                   data-original-title="{% trans "Event" %}">
                    <i class="fas fa-calendar"></i>
                </a>
```

Reply:
I'd prefer this to be an icon to keep more space available for the title itself. Moreover, the href was not quoted. And there was some margin to the right that shouldn't be there I think.

<img width="503" alt="Screenshot 2021-04-01 at 20 03 07" src="https://user-images.githubusercontent.com/1799914/113336109-28f33580-9326-11eb-9673-08ee073f8f3c.png">